pkg/smartcontract/manifest/standard/nep24.go

```diff
@@ -0,0 +1,39 @@
+package standard
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+)
+
+// Nep11WithRoyalty is a NEP-24 Standard for NFT royalties.
+var Nep11WithRoyalty = &Standard{
+	Base: Nep11Base,
```

Comment:
I'm not sure is it correct to put Nep11Base as a base to NEP24, because of two things:
1. The way how `dropStdMethods` works, it will conflict with NEP-11 related code in `rpcbindings.Generate`.
2. Because base for NEP24 is actually either `Nep11NonDivisible` or `Nep11Divisible`.

Think about it, may be it's better to remove this Base and adjust logic of `Generate`  or even to add `Required` field to `manifest.Standard`: if some standard is included into `Required` list then contract should comply both `Required` and current standard.